sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class LogsHelperTests
+    {
+        [Fact]
+        public void MessageIsSetToFormattedMessageWhenIncludeFormattedMessageIsSet()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.IncludeFormattedMessage = true;
+                    options.AddInMemoryExporter(logRecords);
+                });
+                builder.AddFilter(typeof(LogsHelperTests).FullName, LogLevel.Trace);
+            });
+
+            var logger = loggerFactory.CreateLogger<LogsHelperTests>();
+
+            string log = "Hello from {name} {price}.";
+            logger.LogInformation(log, "tomato", 2.99);
+
+            var properties = new ChangeTrackingDictionary<string, string>();
+            var message = LogsHelper.GetMessageAndSetProperties(logRecords[0], properties);
+
+            Assert.Equal("Hello from tomato 2.99.", message);
+            Assert.True(properties.TryGetValue("OriginalFormat", out string value));
+            Assert.Equal(log, value);
+            Assert.True(properties.TryGetValue("name", out string name));
+            Assert.Equal("tomato", name);
+            Assert.True(properties.TryGetValue("price", out string price));
+            Assert.Equal("2.99", price);
+        }
+
+        [Fact]
+        public void MessageIsSetToOriginalFormatWhenIncludeFormattedMessageIsNotSet()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
```

Comment:
Minor, for consistency:
```suggestion
            var logRecords = new List<LogRecord>();
```